card/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'http://rubygems.org'
+
+gemspec
+
+#gem 'card', :path=>'.'
```

Comment:
I'm not sure that I meant to check this in, but I do wan't some things to work from inside the gem dir (like rake?).  Maybe this should be only one or both of the other two.

Opinions on that?